### ISSUE TITLE
Comment without liking?

### ISSUE THREAD

**Issue by ghost** [InstaPy/InstaPy#1836]
Is it possible to comment without liking the picture?
Thanks!

**Comment by timmoh** [comment]
yes => https://github.com/timgrossmann/InstaPy#commenting
but this forum/bugtracker is for issues please use the slack channel

**Comment by ghost** [comment]
From my understanding that function comments on one picture every x pictures that have been liked. Am I wrong?

**Comment by uluQulu** [comment]
hi @ffext,
Already existing `comment_by_locations` feature is just for that, likes are disabled for this method and you can easily rewrite any existing feature to disable likes (_just remove like loops and modify iterations_)...
You may also, want to try this, too:
```python
session.set_do_like(enabled=True, percentage=0)
session.set_do_comment(enabled=True, percentage=100)
session.set_comments(comments=commentsShuffled1, media='Photo')
```
The `like_percentage` above,  currently, is only used with for `interact_by_users` feature and in there, it tries to send comment only after like sent, so it makes no sense.

Well I see no point in commenting without likes (_unless you want to send negative comments xD_), but in the future `like_percentage` should be supported by other features too cos it is a useful parameter.

**Comment by ghost** [comment]
Sorry @uluQulu I don't understand your answer. comment_by_locations enables commenting posts based on location, not tags.

**Comment by uluQulu** [comment]
@ffext,
You asked:
>"Is it possible to comment without liking the picture?"

and it is a general question
